pkg/collector/corechecks/snmp/internal/report/report_metrics_test.go

```diff
@@ -329,6 +334,37 @@ func Test_metricSender_reportMetrics(t *testing.T) {
 				{"[DEBUG] reportScalarMetrics: report scalar: error getting scalar value: value for Scalar OID `1.2.3.4.5` not found in results", 1},
 			},
 		},
+		{
+			name: "report constant metric",
+			metrics: []checkconfig.MetricsConfig{
+				{Symbols: []checkconfig.SymbolConfig{{Name: "constantMetric", ConstantValueOne: true}}, MetricTags: checkconfig.MetricTagConfigList{
+					{
+						Tag:    "status",
+						Column: checkconfig.SymbolConfig{Name: "status", OID: "1.2.3.4"},
+					},
+				}},
+			},
+			values: &valuestore.ResultValueStore{
+				ColumnValues: map[string]map[string]valuestore.ResultValue{
+					"1.2.3.4": {
+						"5.6.7": valuestore.ResultValue{
+							Value: float64(1),
+						},
+						"5.6.8": valuestore.ResultValue{
+							Value: float64(2),
+						},
+						"5.6.9": valuestore.ResultValue{
+							Value: float64(1),
+						},
+					},
+				},
+			},
+			expectedMethod:     "Gauge",
+			expectedMetricName: "snmp.constantMetric",
+			expectedValue:      float64(1),
+			expectedTags:       []string{"status:1"},
+			expectedSubMetrics: 3,
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Can we assert all 3 metrics instead?

You can possibly create a expectedMetrics struct here https://github.com/DataDog/datadog-agent/pull/16547/files#diff-a04138749aca08251bdd24d73eb71c99bbb4f12e1d8e82ddcab7f4c9bf5ec3baR315-R325

and then loop during the assertion. WDYT?